tests/functional/aws-node-sdk/test/multipleBackend/get/getAwsVersioning.js

```diff
@@ -217,12 +217,8 @@ function testSuite() {
                     versionIds.push(...ids);
                     next();
                 },
-                next => getAndAssertResult(s3, {
-                    bucket, key, body: data[4],
-                    versionId: versionIds[4],
-                    expectedVersionId: versionIds[4],
-                },
-                    next),
+                next => getAndAssertResult(s3, { bucket, key, body: data[4],
```

Comment:
why the change?

Reply:
the test is supposed to be for getting the correct version without the version id, but actually tested getting with version id. I think no one caught this small detail when reviewing the original PR.